### DISCUSSION TITLE
Remove @nrwl/nx and return back to normal angular cli

### DISCUSSION THREAD

**Discussion by Wayne-Mather** [nrwl/nx#8353] [General]
Hi,

There are plenty of docs on how to add nx to an angular project, or start from nx with the angular schematics. My question is how do I  go back to just a standard angular cli application?

I know I can just start a new project and copy the required files over, but was wondering if there is an easier way to just move back to normal angular schematics?

I used NX because I thought the project would have more than 1 project, but since it never will, I don't need the overhead of the nx upgrade complexities. It's an NX v9 workspace that I want to take to a v13 standard angular app.

Any information on how to achieve this easily would be appreciated.

**Comment by aaronscribner** [comment]
This would be useful, but seems like they do not want to give directions on removing it.

**Comment by Ser-Custodio** [comment]
Hi, did you ever find out how to do it ?

**Reply by Wayne-Mather** [reply]
Was a long time ago and dont work on that project anymore. But yes, removed nx and I think just created new project and moved the files over, then upgraded the project

**Reply by Ser-Custodio** [reply]
Thanks, I'll try to do the same here

**Reply by Wayne-Mather** [reply]
Maybe there is an easy way, been that long. What version on nx are you using? Might have a go sometime to see how easy it is to remove the schematics

**Reply by Ser-Custodio** [reply]
I'm using the latest (20.6.2) but finding to many bugs so since I only have 1 project inside guess I'm gonna do as you did. Not finding anything in the docs for that :D

**Reply by Wayne-Mather** [reply]
I might have got it. Again, have backups to validate this does work..

1. Create a new angular project: ng new myapp
2. In the nx repo move nx.json to nxjson.old
3. Copy angular.json from myapp into nx repo
4. Copy tsconfig* from myapp into nx repo
5. Move apps\xxx\src into main nx folder
6. ng build should now work
7. Delete the apps directory, .nx directory and the nx json file
8. You should now have a angular repo that works as expected
9. You can also remove the myapp folder as not needed anymore

The reason I copied the config files from the angular project instead of src was to ensure they are for angular incase nx has anything that could break something.

Again, was for an nx monorepo with a single angular project in it using nx 20.6.2

**Reply by Ser-Custodio** [reply]
Thank you, gonna give that a try 👌

**Reply by Wayne-Mather** [reply]
@Ser-Custodio how did it go? Did this work for you?

**Reply by Ser-Custodio** [reply]
Didn't try yet, need aprouval from rest of the team. I'll let you know if is accepted

**Reply by jcannon98188** [reply]
I'm about to undertake this same process this week, so I will report back on how it goes, but this does match what I would expect to do as part of this.

**Reply by Wayne-Mather** [reply]
@jcannon98188 How did it go? Work for you?

**Reply by jcannon98188** [reply]
Happy to report we successfully were able to do the ejection with almost no issues. There was some goofiness on our end from bad code choices (made before I got to the codebase I swear!) but the ejection process itself was smooth.

**Reply by Wayne-Mather** [reply]
@jcannon98188 Thats good to hear. Was it the same steps I mentioned above? Might be good to put some information here and then maybe we just close this thread as we now have a working method. I think at the time I created it was an older version of NX and it possibly did not use the angular json file but an nx.json from memory. I could be wrong - was 2022 after all!
